docs/source/02_get_started/04_new_project.md

```diff
@@ -13,18 +13,18 @@ kedro new
 You will be asked to enter each of the following variables in turn. Once you have entered text for the first option (the project's name), you will be offered a default choice for the other options:
 
 ```eval_rst
-+------------------------+---------------------+----------------------------------------------------------------------------+
-| Option                 | Example             | Description                                                                |
-+========================+=====================+============================================================================+
-| :code:`project_name`   | :code:`Get Started` | A human-readable name for your new project                                 |
-+------------------------+---------------------+----------------------------------------------------------------------------+
-| :code:`repo_name`      | :code:`get-started` | Directory that holds your project repository                               |
-+------------------------+---------------------+----------------------------------------------------------------------------+
-| :code:`python_package` | :code:`get_started` | A name for the Python package name in your project (short, all-lowercase)  |
-+------------------------+---------------------+----------------------------------------------------------------------------+
-| :code:`include_example`| :code:`Y/n`         | Confirms or rejects the inclusion of the Iris example code                 |
-|                        |                     | Your new project template contains a small example if you enter :code:`Y`  |
-+------------------------+---------------------+----------------------------------------------------------------------------+
++------------------------+-------------------------+----------------------------------------------------------------------------+
+| Option                 | Example                 | Description                                                                |
++========================+=========================+============================================================================+
+| :code:`project_name`   | :code:`Getting Started` | A human-readable name for your new project                                 |
```

Comment:
I very specifically removed gerunds from the entirety of the Kedro documentation: getting started -> get started.

I am not enthusiastic about adding them back in unless there's a great reason to do so?

Reply:
My aim was just to make the 'Create a new project' page consistent with the 'Iris dataset example project' page. (The  has getting-started), and it was 50-50 which convention I chose. I will change them so there is only get-started.